jest.setup.ts

```diff
@@ -1,4 +1,5 @@
 import {GlobalWithFetchMock} from 'jest-fetch-mock';
+import 'abortcontroller-polyfill/dist/abortcontroller-polyfill-only';
```

Comment:
The [`AbortController`](https://developer.mozilla.org/en-US/docs/Web/API/AbortController) object is not defined in a standard node runtime environment so we use a polyfill so that the unit tests can still run successfully.

Reply:
![Screenshot from 2019-11-07 07-24-07](https://user-images.githubusercontent.com/45688129/68388896-a7e33f00-012f-11ea-869d-0cce02c92b21.png)

you don't think it will backfire on us ?

Reply:
The browser compatibility is the same as for `fetch` that pretty much is the core upon which we build the platform-client, so its highly improbable that someone would use the `platform-client` without having the `AbortController` polyfill.

Perhaps we could specify in the README.md that our package doesn't include polyfills for `fetch` and the `AbortController` and maybe provide some links to those packages.

Reply:
Added a [`Compatibility`](https://github.com/coveo/platform-client/blob/ADUI-4886-abort-get-calls/README.md#compatibility) section to the README.